frontend/src/components/Sidebar/prepareRoutes.ts

```diff
@@ -240,6 +230,16 @@ function prepareRoutes(
         },
       ],
     },
+    {
+      name: 'crds',
+      label: t('glossary|Custom Resources'),
```

Comment:
Please add icon: `'mdi:puzzle',` here so we get a nice icon for CRs. The icon is this one: https://icon-sets.iconify.design/mdi/puzzle/
The contents of the SVG should be copied into the icon cache at `frontend/src/components/App/icons.ts`, with the format similar to those already there. So this icon works fine offline.